sdcm/cluster_baremetal.py

```diff
@@ -0,0 +1,150 @@
+import logging
+
+import cluster
+from . import wait
+
+logger = logging.getLogger(__name__)
+
+BASE_NAME = 'db-node'
+LOADER_NAME = 'loader-node'
+MONITOR_NAME = 'monitor-node'
+
+
+class NodeIpsNotConfiguredError(Exception):
+    pass
+
+
+class PhysicalMachineNode(cluster.BaseNode):
```

Comment:
We can use this to connect any physical machine, virtual machine, or something which has available ip.
How about replace `PhysicalMachine` with `LiveMachine`?

Reply:
There are some preparation need to be done on the machine:
- create scylla-test user, if we use hardcode username
- set sudo without password
- ssh key auth
- firewall?
- more?

At least we need to write it to doc.

Reply:
Physical machine and an existing environment are still different concepts. Some things are differ for physical and virtual machine, e.g. restart or destroy.

Reply:
We have a plan to support "existing cluster" (Reuse cluster).
This is lower priority right now, but the foundations for it can be laid in this PR.

Existing cluster behaviour require few more definitions and stages.
(e.g. to populate the cluster with the "exact" state it was before the test).

Reply:
Ok, let's continue use `PhysicalMachine`.